examples/ResNet/ResNet.cpp

```diff
@@ -371,8 +379,10 @@ const wnn::Operand ResNet::LoadNHWC(const wnn::GraphBuilder& builder, bool softm
 
     // Block 3
     const wnn::Operand bottleneck8 = BuildNhwcBottlenectV2(builder, bottleneck7, {"3", "1"}, true);
-    const wnn::Operand bottleneck9 = loop(builder, bottleneck8, 2);
-    const wnn::Operand bottleneck10 = BuildNhwcBottlenectV2(builder, bottleneck9, {"3", "23"});
+    uint32_t loopEnd = is50V2 ? 5 : 22;
```

Comment:
There is another parameter need is50V2. So we can remove 101 model tests directly to avoid additional work, if you don't want to keep the parameter is50V2. @fujunwei

Reply:
It also should be configured in Reset.h as member variable.

Reply:
As discussed with you, I removed nhwc resnet101 model tests, PTAL, thanks! @fujunwei